src/include/kernel/tokens.h

```diff
@@ -71,6 +71,7 @@
 /* Token retired with ABI 3.2, do not use for new capabilities
  * #define SOF_TKN_COMP_PRELOAD_COUNT		403
  */
+#define SOF_TKN_COMP_CORE_ID			404
```

Comment:
Is this token missing from kernel or FW. I'm not following this should have been part of multi core component added by @lyakh

Reply:
There's a PR for kernel by @lyakh .

Reply:
@lyakh can you comment, has your kernel PR been merged ? Which one is it ?